core50.md

```diff
@@ -0,0 +1,237 @@
+# Apache MyFaces Core 5.0
+
+Implementation of the Jakarta Faces 5.0 specification.  This is still in development and not yet released. 
+
+## Requirements
+* Java 21
+* Servlet 6.1+
+* EL 6.0
+* CDI 4.1
+* JSTL 3.0 (optional)
+* BV 3.1 (optional)
```

Comment:
Updated to EE11. I believe 21 is the required runtime based on my current understanding.